no-parser/package.json

```diff
@@ -3,5 +3,9 @@
   "private": true,
   "main": "../dist/styled-components-no-parser.cjs.js",
   "module": "../dist/styled-components-no-parser.es.js",
-  "jsnext:main": "../dist/styled-components-no-parser.es.js"
+  "jsnext:main": "../dist/styled-components-no-parser.es.js",
+  "browser": {
+    "../dist/styled-components-no-parser.es.js": "../dist/styled-components-no-parser.browser.es.js",
```

Comment:
when i have last experimented with this, those `browser` entries need to be put in the root package.json, not in the "proxying" ones - that need to be checked (preferrrably on the real example, will be hard to checkout any docs on this)

i **think** the algorithm just traverses up from the module for the nearest `package.json` to check against browser "rewrites", but when we are in the `dist` directory this is the root `package.json` - when this happens `browser-resolve` doesnt really know that we got the `dist` by proxy `package.json`, it knows only the requested module id and the absolute path of the requesting module